classes/autoloader.php

```diff
@@ -41,7 +41,17 @@ public function __construct(array $namespaces = array(), array $namespaceAliases
 			$this->addNamespaceAlias($alias, $target);
 		}
 
-		foreach ($classAliases ?: array('atoum' => __NAMESPACE__ . '\test', __NAMESPACE__ => __NAMESPACE__ . '\test') as $alias => $target)
+		$defaultAliases = array(
+			'atoum' => __NAMESPACE__ . '\test',
+			__NAMESPACE__ => __NAMESPACE__ . '\test'
+		);
+
+		// Remove BC break for PHP_VERSION lower than 7.1.0 void is configured to be an alias of blank.
+		if (version_compare(PHP_VERSION, '7.1.0alpha2') < 0) {
```

Comment:
Unfortunately it's not possible if I remove the condition it raise a fatal Error

```
Fatal error: Cannot use 'mageekguy\atoum\report\fields\runner\tests\void' as class name as it is reserved in /Users/ashgenesis/dev/atoum/classes/autoloader.php on line 244
```